moleculemaker/chemscraper-frontend
# feat: allow sorting by Number of Atoms

## Problem
We would like to display/sort by the number of atoms in each molecule

Fixes #76 

Relates to https://github.com/moleculemaker/mmli-backend/pull/30

## Approach
* Added new option to sort by "Number of Atoms" - this is now returned from the API as `atom_count`

## How to Test
This is currently deployed on [chemscraper-frontend-staging](https://argocd.devops.ncsa.illinois.edu/applications/argocd/chemscraper-frontend-staging)

1. Navigate to https://chemscraper.frontend.staging.mmli1.ncsa.illinois.edu/results/02b0034a-4335-4823-8d5d-915b54173995
2. Click on the Sort options at the top-right of the table (default: "Location in PDF")
    * You should see a new option here for "Number of Atoms"
3. Choose to sort by "Number of Atoms"
    * You should see the table reorder itself to sort by `atom_count`
4. Click the blue button on the popover to change whether to sort by ascending or descending
